### PR TITLE
keep folio field for staff

### PR DESCRIPTION
**Pull request by garyfan1**
*Issue #:* /bcgov/entity#12074

*Description of changes:*
keep folio or reference number text filed for staff.
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by cameron-eyds — COMMENTED** [review]
Looks good so far.
Will need to update the unit testing to verify it's ALWAYS shown. 

Also, see comments in our Assets channel, as we've now moved to node version `16.14.2`, all the front end contributors will need to be on the same version.

**Review by dimak1 — CHANGES_REQUESTED** [review]
Agree with Cameron. Please add some tests.

File: ppr-ui/src/components/search/SearchBar.vue [hunk 1]
```diff
@@ -40,7 +40,7 @@
           </v-tooltip>
         </div>
       </v-col>
-      <v-col v-if="!isStaffBcolReg && !isStaffSbc" align-self="end" cols="3">
+      <v-col align-self="end" cols="3">
```
Comment (cameron-eyds) [inline ppr-ui/src/components/search/SearchBar.vue:43]: Was there further commits here to render for specific types of Staff or is the requirement to display for all, client and staff?

**Review by cameron-eyds — APPROVED** [review]
Thanks for updating the test.
Looks good to me, just the one question. 
You mentioned previously that it might be rendered for only specific types of staff but if no further change is required, you are g2g 👍

File: ppr-ui/src/components/search/SearchBar.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/search/SearchBar.vue:43]: From the story, the requirement says it should appear for all staff users (in addition to clients). Right @garyfan1 ?

Comment (garyfan1) [reply ppr-ui/src/components/search/SearchBar.vue:43]: I checked with Melissa and the field is shown in all cases.

**Review by dimak1 — APPROVED** [review]

**Pull request merged**